javalanglib/src/main/scala/java/lang/Float.scala

```diff
@@ -65,21 +66,81 @@ object Float {
   @inline def toString(f: scala.Float): String =
     "" + f
 
+  def toHexString(f: scala.Float): String = {
```

Comment:
[JavaDoc of `Float.toHexString`](https://docs.oracle.com/javase/8/docs/api/java/lang/Float.html#toHexString-float-)